core/compiler.cpp

```diff
@@ -447,6 +447,15 @@ torch::jit::Module CompileGraph(const torch::jit::Module& mod, CompileSpec cfg)
         auto engine = conversion::ConvertBlockToEngine(g->block(), cfg.convert_info, static_params);
         AddEngineToGraph(new_mod, new_g, engine, cuda_device);
       }
+      for (auto input : new_g->block()->inputs()) {
+        LOG_DEBUG("input name:" << input->debugName());
+        if (input->debugName().find(".") != std::string::npos) {
+          auto pos = input->debugName().find(".");
+          auto newName = input->debugName().replace(pos, 1, "_");
```

Comment:
This seems hacky, @bowang007 thoughts? We might just be doing something wrong upstream causing these issues.

Reply:
what happens is that when there is any fallback we will copy the Metadata of the inputs of the lowered graph (include debugname()) to the inputs of the new graph through this [line](https://github.com/pytorch/TensorRT/blob/a3432e250e7fe35618607ea95d632a1833d225c4/core/compiler.cpp#L237) As a result, the inputs' names' are same with the Lowered Graph inputs, which has some values as x.1, these names cannot be deserialized by torch.jit.load(). This explains why we are not getting this error when there is no fallback, since all the inputs are substituted by input_0, input_1 when there is no fallback. 
Another solution to this error would be simply deleting the mapping [here](https://github.com/pytorch/TensorRT/blob/a3432e250e7fe35618607ea95d632a1833d225c4/core/compiler.cpp#L237). I did some tests locally which could help resolve this issue. However, not sure if this deletion would induce other issues, will run the test cases to ensure it works.